modules/oe_theme_helper/config/schema/oe_theme_helper.schema.yml

```diff
@@ -0,0 +1,10 @@
+image.effect.retina_image_scale:
+  type: image_size
```

Comment:
Since you are extending from the scale image effect, you can just call it type `image.effect.image_scale` and inherit its properties. And add your own `multiplier` property to the definition.

Reply:
I am extending the scale image effect but I'm also creating a new plugin with a separate id. Correct me if I'm wrong but if I use the "image.effect.image_scale" type then the multiplier property would apply to configuration using the "image_scale" id.

Reply:
config schema "types" extend one another. So `image.effect.image_scale` extends `image_size` and adds its own config fields. You can extend from `image.effect.image_scale` and add yours (the multipler0